src/bridge/views/Network.js

```diff
@@ -34,9 +35,7 @@ class Network extends React.Component {
     if (network === NETWORK_NAMES.ROPSTEN) {
       setNetworkType(network);
 
-      const endpoint = `https://ropsten.infura.io/v3/${
-        process.env.REACT_APP_INFURA_ENDPOINT
-      }`;
+      const endpoint = `https://ropsten.infura.io/v3/${process.env.REACT_APP_INFURA_ENDPOINT}`;
```

Comment:
Again, there may be some `prettier` weirdness here.

Reply:
fixed my editor 👍